modules/virt-deleting-virt-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/install/uninstalling-virt.adoc
+
+[id="virt-deleting-virt-cli_{context}"]
+= Deleting the {VirtProductName} using the CLI
+
+You can delete {VirtProductName} by using the CLI.
+
+.Prerequisites
+
+- `oc` command installed on workstation.
```

Comment:
~~~
* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
~~~

Reply:
Updated the content.